### DISCUSSION TITLE
My Custom Tile Integrating with DT Metrics

### DISCUSSION THREAD

**Discussion by oldvill** [DiscipleTools/disciple-tools-theme#1160] [Ideas and Requests]
I would love to see information from my I custom tile (within Contacts) show up in the Metrics section.
Thanks for considering this integration. 
I 'think' I attached a screenshot of my custom tile. 
![Screen Shot 2021-02-01 at 11 53 25 AM](https://user-images.githubusercontent.com/25636245/106449396-3a35d800-6484-11eb-8dfb-aa999e3785f6.png)

**Comment by corsacca** [comment]
@oldvill what kind of metrics are you looking to see?  ex: bibles received by month

**Reply by oldvill** [reply]
I would love to see all of those metrics show up on the metrics page. but if I need to prioritize:
1. Requests
2. Translation
3. Contact Preference
I added this tile and have found that our Multipliers really like being able to see at a glance what the contact is desiring. 
As standalone metrics, they might need to be renamed.

**Reply by corsacca** [reply]
Are you referring to the critical path metrics page?

**Reply by oldvill** [reply]
honestly i have not thought of where would be the best placement. the 'Requests' could be part of the critical path but certainly 'Translation' is NOT. 
Instead Project-->Critical Path, it could go under Contacts.

**Reply by corsacca** [reply]
is the  `/metrics/contacts/milestones` chart the kind of chart you are looking for?
There are many different ways to "display" data. The more you can be specific on the kind of charts, on what should be displayed (what on the x axis and what on the y axis), what timeframe, the better someone can try to build something that will fit the request.

**Reply by oldvill** [reply]
YES! exactly like the /metrics/contacts/milestones chart; including the same date range options.
this would fantabulous.
